src/applications/vaos/appointment-list/sass/styles.scss

```diff
@@ -69,6 +69,24 @@
     padding: 24px;
     margin-bottom: 24px;
   }
+
+  &.vaos-appts__card--clickable {
+    list-style: none;
+    padding: 0;
+
+    .vaos-appts__card--clickable-content {
```

Comment:
You could make the functionality in this class with the design system classes: `vads-u-padding--2 medium-screen:vads-u-padding--3 medium-screen:vads-u-margin-bottom--3`

Reply:
@jbalboni makes sense not to add the bloat. I am leveraging the design system classes now. Just take a peek again to verify nothing got asked on us.